test/language/expressions/arrow-function/static-init-await-binding.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-identifiers
+description: The restriction on `await` as a BindingIdentifier does not cross function boundaries
+info: |
+  BindingIdentifier : Identifier
+
+  [...]
+  - It is a Syntax Error if the code matched by this production is nested,
+    directly or indirectly (but not crossing function or static initialization
+    block boundaries), within a ClassStaticBlock and the StringValue of
+    Identifier is "await".
+features: [class-static-block]
+---*/
+
+class C {
+  static {
+    (await => 0);
```

Comment:
Similarly here and in other arrow function tests, `await` is never allowed as an arrow parameter in async function bodies, so shouldn't here either.

Reply:
As mentioned by @syg, `await` should be disallowed here, per https://github.com/tc39/proposal-class-static-block/pull/46